eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf

```diff
@@ -1,113 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is .pt.xlf so I removed it manually

For the strings added to messages it's due to this : https://github.com/schmittjoh/JMSTranslationBundle/pull/427